meson.build

```diff
@@ -373,12 +377,21 @@ else
     libass_for_tools = libass
 endif
 
-if get_option('test')
+if get_option('test').enabled()
```

Comment:
What's up with `.enabled()`? If it is intended to be a boolean option instead of feature, it should be defined as such in `meson_options.txt`.

Reply:
A couple of these have actual dependency interactions, and it didn't make sense for some of the utils to be features while others were booleans.